UPGRADE-8.0.md

```diff
@@ -333,6 +333,41 @@ Security
 Serializer
 ----------
 
+ * Remove escape character functionality from `CsvEncoder`
+
+   *Before*
+   ```php
+   use Symfony\Component\Serializer\Encoder\CsvEncoder;
+
+   // Using escape character in encoding
+   $encoder = new CsvEncoder();
+   $csv = $encoder->encode($data, 'csv', [
+       CsvEncoder::ESCAPE_CHAR_KEY => '\\',
+   ]);
+
+   // Using escape character with context builder
+   use Symfony\Component\Serializer\Context\Encoder\CsvEncoderContextBuilder;
+   
+   $context = (new CsvEncoderContextBuilder())
+       ->withEscapeChar('\\')
+       ->toArray();
+   ```
+
+   *After*
```

Comment:
In another case, I had the before/after be a diff. I feel like it might be worth using this style here.
I fear the file might end up overcrowded if we do these before/after all the time :grimacing: